Makefile

```diff
@@ -1,39 +1,44 @@
 include ./config.mk
 
 TAGNAME ?= default
+
+.PHONY: test-env-up
 test-env-up:
-	@cd ../.. && docker build . -q -f ./tests/acceptance/scripts/Dockerfile.build -t acceptance-test-${TAGNAME}
+	docker build . -q -f ./scripts/Dockerfile.build -t acceptance-test-${TAGNAME}
 
 .PHONY: test-run
 test-run:
-	@docker run -d --name acceptance-test${IMGNAME} -t \
+	@docker run -d --name acceptance-test-${IMGNAME} -t \
       -e AWS_ACCESS_KEY_ID="${AWS_ACCESS_KEY_ID}" \
       -e AWS_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY}" \
-      -v ${ACCESS_KEY_LOCAL}:/go/src/github.com/k3s-io/k3s/tests/acceptance/modules/k3scluster/config/.ssh/aws_key.pem \
-      k3s-automated-${TAGNAME} sh -c 'cd ./tests/acceptance/entrypoint; \
+      -v ${ACCESS_KEY_LOCAL}:/go/src/github.com/rancher/distros-test-framework/config/.ssh/aws_key.pem \
+      acceptance-test-${TAGNAME} sh -c 'cd ./entrypoint; \
     if [ -n "${TESTDIR}" ]; then \
         if [ "${TESTDIR}" = "upgradecluster" ]; then \
             go test -timeout=45m -v ./upgradecluster/... -installVersionOrCommit "${INSTALLTYPE}"; \
         elif [ "${TESTDIR}" = "versionbump" ]; then \
             go test -timeout=45m -v -tags=versionbump ./versionbump/... -cmd "${CMD}" -expectedValue "${VALUE}" \
```

Comment:
can you just delete the comments ? and them in another task we fix the whole makefile

Reply:
Updated

Reply:
i meant the comments bellow , after the file .

Reply:
Not sure if I understand the ask, please advise

Reply:
it was not updated on my end! nw , sorry